.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+-fd
```

Comment:
For future reference, I usually also add `--require spec_helper.rb` so that you don't have to have that require at the top of every spec file, but that may just be my preference.  Don't need to change the PR